lita-jarvis.gemspec

```diff
@@ -25,10 +25,11 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "mbox"
   spec.add_runtime_dependency "cabin"
   spec.add_runtime_dependency "open4"
-  #spec.add_runtime_dependency "lita-hipchat"
+  spec.add_runtime_dependency "lita-hipchat"
```

Comment:
I did not know about that, sorry.  I was looking for a way to run it locally.  Will fix.